website/docs/r/security_partner_provider.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_security_partner_provider"
+description: |-
+  Manages a Security Partner Provider.
+---
+
+# azurerm_security_partner_provider
+
+Manages a Security Partner Provider.
```

Comment:
Could we elaborate on what a SPP is and what it is applied to?

Reply:
updated